bin/pt-online-schema-change

```diff
@@ -8562,7 +8562,7 @@ sub main {
          #   at /Users/daniel/p/pt-osc-2.1.1/lib/PerconaTest.pm line 559.
          #                   ''
          #   doesn't match '(?-xism:Failed to find a unique new table name)'
-         my $original_error = $EVAL_ERROR;
+         #my $original_error = $EVAL_ERROR;
```

Comment:
Yes. It actually is a fix for another bug that made debugging this fix difficult. Will remove it as it will get merged anyway.